compiler/test/fail_compilation/ctor_attr.d

```diff
@@ -1,7 +1,7 @@
 /*
 TEST_OUTPUT:
 ---
-fail_compilation/ctor_attr.d(26): Error: none of the overloads of `this` are callable using argument types `(int)`
+fail_compilation/ctor_attr.d(26): Error: none of the overloads of `this` are callable using a mutable object
```

Comment:
This is not entirely right, since there is a mutable constructor, but for `string` and not for `int`. Perhaps you can still include the argument types in the message?

Reply:
Similar issue fixed in #15836